plugins/hosts/linux/cap/nfs.rb

```diff
@@ -102,7 +102,8 @@ def self.nfs_cleanup(id)
 
           # Use sed to just strip out the block of code which was inserted
           # by Vagrant
-          system("cp /etc/exports $TMPDIR && #{sudo_command}sed -r -e '\\\x01^# VAGRANT-BEGIN:( #{user})? #{id}\x01,\\\x01^# VAGRANT-END:( #{user})? #{id}\x01 d' -ibak $TMPDIR/exports ; cp $TMPDIR/exports /etc/exports")
+          tmp = ENV["TMPDIR"] || ENV["TMP"] || "/tmp"
+          system("cp /etc/exports '#{tmp}' && #{sudo_command}sed -r -e '\\\x01^# VAGRANT-BEGIN:( #{user})? #{id}\x01,\\\x01^# VAGRANT-END:( #{user})? #{id}\x01 d' -ibak '#{tmp}/exports' ; cp '#{tmp}/exports' /etc/exports")
```

Comment:
Hi @kkopachev 

Could you please open a new issue for that? It's easy to lose track of that in comments. Thanks! :smile: